lega/ingest.py

```diff
@@ -59,8 +60,15 @@ def work(fs, inbox_fs, data):
 
     # Insert in database
     file_id = db.insert_file(filepath, user_id)
+
     data['file_id'] = file_id  # must be there: database error uses it
 
+    # we will be working with sha256 only and let us see if we get in the message this for now
+    # if this is not present we will calculate it
+    file_checksum = []
+    if 'encrypted_checksums' in data:
```

Comment:
Did you agree in the hookup on using the actual implementation (encrypted_checksums) for this rather than what I understood as agreed (encrypted_integrity)?

Otherwise, could we support both names for future safety?

Reply:
we did not agree on that, that is the main issue. but just quoting what they say:
> we send back the message content we receive

meaning if our inbox sends that then it is ok, and even so if we don't get that we calculate the checksum below in the code, so no need to support both names. Once they decide on the interface we will use that. I just reused what is in their source code as well: https://github.com/EGA-archive/LocalEGA-inbox/blob/master/src/mq-notify.c#L315